src/web/app/pages-admin/admin-sessions-page/admin-sessions-page.component.ts

```diff
@@ -3,29 +3,13 @@ import moment from 'moment-timezone';
 import { HttpRequestService } from '../../../services/http-request.service';
 import { StatusMessageService } from '../../../services/status-message.service';
 import { TimezoneService } from '../../../services/timezone.service';
-import { FeedbackSessionStats } from '../../../types/api-output';
+import { FeedbackSessionStats, OngoingSession, OngoingSessions } from '../../../types/api-output';
 import { ErrorMessageOutput } from '../../error-message-output';
 
-interface OngoingSession {
-  sessionStatus: string;
-  instructorHomePageLink: string;
-  startTime: number;
-  endTime: number;
-  creatorEmail: string;
-  courseId: string;
-  feedbackSessionName: string;
+interface OngoingSessionInternal extends OngoingSession {
```

Comment:
It is better to call it `OngoingSessionModel` because it is something you will update and sync between template and component.
```
interface OngoingSessionModel {
    ongoingSession: OngoingSession,
    responseRate?: string
}
```

Reply:
Would it be ok to still leave it as extending ongoingSession? I'm not sure how to fix the UI otherwise, converting it to a variable causes the getResponses() to fail. My current latest commit implements this with map, but on the page clicking show responses does nothing.